lms/djangoapps/mobile_api/social_facebook/groups/tests.py

```diff
@@ -0,0 +1,171 @@
+        member_id = '1234,,,,5678,,'
+        response = invite_to_group(self, group_id, member_id)
+        self.assertEqual(response.status_code, 400)
+        
+    def test_invite_single_member_malformed_member_id_2(self):
+        group_id = '756869167741019'
+        member_id = 'this00is00not00a00valid00id'
+        response = invite_to_group(self, group_id, member_id)
+        self.assertEqual(response.status_code, 400)
+
+    def test_invite_single_member_no_member_id(self):
+        group_id = '756869167741019'
+        member_id = ''
+        response = invite_to_group(self, group_id, member_id)
+        self.assertEqual(response.status_code, 400)
+
+    def test_invite_single_member_malformed_member_id_3(self):
+        group_id = '756869167741019'
+        member_id = '1234,abc,5678'
+        response = invite_to_group(self, group_id, member_id)
+        self.assertEqual(response.status_code, 400)
+
+    def test_invite_single_member(self):
+        group_id = '756869167741019'
+        member_id = '10154831816670300'
+        # Note that the memeber must be a member of the app and not a member 
+        # of the group in order to be able to join the group.
+        response = invite_to_group(self, group_id, member_id)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('success' in response.data[member_id])  # pylint: disable=E1103
+        # Remove user from the group
+        remove_from_group(self, group_id, member_id)
+
+    def test_invite_multiple_members_successfully(self):
+        member_ids = '366785273488903,939400156088941,10154831816670300'
+        group_id = '756869167741019'
+        response = invite_to_group(self, group_id, member_ids)
+        self.assertEqual(response.status_code, 200)
+        for member_id in member_ids.split(','):
+            self.assertTrue('success' in response.data[member_id])  # pylint: disable=E1103
+        # Remove the members added
+        for member_id in member_ids.split(','):
+            remove_from_group(self, group_id, member_id)
+
+    def test_invite_multiple_members_unsuccessfully(self):
+        # Add a single user
+        member_id = '10154831816670300'
+        group_id = '756869167741019'
+        response = invite_to_group(self, group_id, member_id)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('success' in response.data[member_id])  # pylint: disable=E1103
+        # Invite three members, two who are not in the group and one that is a member already.
+        member_ids = '366785273488903,939400156088941,10154831816670300'
+        response = invite_to_group(self, group_id, member_ids)
+        self.assertEqual(response.status_code, 200)
+        for member_id in '366785273488903,939400156088941'.split(','):
+            self.assertTrue('success' in response.data[member_id])
+        self.assertTrue('(#4001) User is not eligible to be added to group' in response.data['10154831816670300'])
```

Comment:
Nit: In our test code, we generally prefer using the most specific Assertion function so our test error messages are most accurate.  To abide by this, all cases of `self.assertTrue(x in y)` should be replaced with `self.assertIn(x, y)`.